google/cloud/aiplatform/__init__.py

```diff
@@ -91,4 +92,5 @@
     "TextDataset",
     "TimeSeriesDataset",
     "VideoDataset",
+    "data_types",
```

Comment:
This name needs to be qualified a bit more. `aiplatform.data_types` seems too generic. Perhaps, `automl_tabular_column_types`.